merkle-tree/indexed/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 [dependencies]
 ark-ff = "0.4"
+borsh = { version = "0.10" }
```

Comment:
why don't you use anchor deserialize?
 I have had dependency conflicts trying to use both borsh and anchor

Reply:
Very funny reason - adding anchor as a dependency changes the behavior of `Result` - you have to stick to Anchor's `#[error_code]` and can't use `thiserror`.

Dependency conflicts happen if you try to use too old or too new borsh, but using 0.10.3 works fine.

Reply:
ah I believe I was using 0.8.3